src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -60,7 +65,7 @@ class SearchMenu extends React.Component {
         id="searchmenu"
         icon={icon}
         isOpen={this.props.isOpen}>
-        {this.makeForm()}
+        {this.makeForm(searchAction, searchAffiliate)}
```

Comment:
Is it better to inject as such from the top-most component, or is it okay to define these values within the `makeForm()` function?

Reply:
Could put them on the component state in the constructor so this helper is not run every time the component renders. Not sure why `makeForm()` is a function and not just inlined jsx but you don't have to refactor that.